.gitignore

```diff
@@ -63,16 +66,11 @@ extensions/core/
 experimental/
 
 # nupic bindings swig libraries
-nupic/bindings/*engine_internal.*
-nupic/bindings/*algorithms.*
-nupic/bindings/*iorange.*
-nupic/bindings/*math.*
-nupic/bindings/*.cxx
-nupic/bindings/proto/*.capnp
+nupic/bindings/
+~nupic/bindings/*.i
```

Comment:
what is `~nupic`? I'm not convinced that I want this hidden from me if it gets created